src/zh/baozimhorg/src/eu/kanade/tachiyomi/extension/zh/baozimhorg/GoDaManhua.kt

```diff
@@ -23,6 +23,8 @@ class GoDaManhua : GoDa("GoDa漫画", "", "zh"), ConfigurableSource {
 
     override val baseUrl: String
 
+    override fun headersBuilder() = super.headersBuilder().add("Referer", "$baseUrl/").add("origin", "$baseUrl/")
```

Comment:
Only the `Referer` header has a trailing slash, `Origin` does not.
Casing might matter for fingerprinting purposes if using HTTP/1.1.
```suggestion
    override fun headersBuilder() = super.headersBuilder().add("Referer", "$baseUrl/").add("Origin", baseUrl)
```